examples/http_get_bearssl/http_get_bearssl.c

```diff
@@ -201,6 +229,15 @@ void http_get_task(void *pvParameters)
         br_sslio_init(&ioc, &sc.eng, sock_read, &fd, sock_write, &fd);
         printf("done.\r\n");
 
+        /* FIXME: set date & time using epoch time precompiler flag for now */
+        provisional_time = CONFIG_EPOCH_TIME + (xTaskGetTickCount()/configTICK_RATE_HZ);
+        xc.days = (provisional_time / CLOCK_SECONDS_PER_DAY) + 719528;
+        xc.seconds = provisional_time % CLOCK_SECONDS_PER_DAY;
```

Comment:
@UncleRus I extended the timer code to continuously progress time. From a functional perspective I believe the code is ready to be merged now.